### PR TITLE
Graphql Error Management

### PR DESCRIPTION
**Pull request by jycouet**

### CODE REVIEW

File: projects/core/graphql.ts [hunk 1]
```diff
@@ -426,26 +476,34 @@ Select a dedicated page.`,
...
-              arg1.input,
-            )
-          },
-        )
+                arg1.input,
+              )
+            },
+          )
+        } catch (error) {
+          console.log(`coucou`);
```
Comment (noam-honig) [inline projects/core/graphql.ts:504]: ??

**Review by noam-honig — COMMENTED** [review]
Is there anything you would like me to do - or is it complete and ready for merge from your end?

File: projects/core/graphql.ts (hunk 1 above)
Comment (jycouet) [reply projects/core/graphql.ts]: Oups!
My debug log is visible ^^

**Comment by jycouet** [comment]
> Is there anything you would like me to do - or is it complete and ready for merge from your end?

I started the POC for `create` (here), but it needs for update & delete before merging.
I would need your help on: 
- Throwing at the resolver level
- General "clientMutationId" solution

Let's chat later

File: projects/tests/graphql.spec.ts [hunk 2]
```diff
@@ -369,7 +556,86 @@ describe("graphql-connection", () => {
...
+          }
+        }
+      }
+    }`)
+    expect(result).toMatchInlineSnapshot(`
+      {
+        "data": {
+          "createTask": {
+            "error": {
+              "message": "The Title: Too short",
```
Comment (noam-honig) [inline projects/tests/graphql.spec.ts:626]: Why as an array and not as fields in a similar way that it looks in regular rest api?

Comment (jycouet) [reply projects/tests/graphql.spec.ts:626]: We talked about it, but just to resolve the point here:
We don't wan't people to select each field (in the graphql query) to see if there is an error, that's why it's an array.

**Pull request merged**